docs/source/api/v3/servercheck.rst

```diff
@@ -31,7 +31,33 @@ Fetches identifying and meta information as well as "check" values regarding all
 
 Request Structure
 -----------------
-No parameters available.
+.. table:: Request Query Parameters
+
+	+-----------+------------------------------------------------------------------------------------+
+	| Name      | Description                                                                        |
+	+===========+====================================================================================+
+	| id        | Return only Servers with this integral, unique identifier (id)                     |
+	+-----------+------------------------------------------------------------------------------------+
+	| name      | Return only Servers with this host_name                                            |
+	+-----------+------------------------------------------------------------------------------------+
+
+.. code-block:: http
+	:caption: Request Example with ``name`` query param
+
+   GET /api/4.0/servercheck?name=edge HTTP/1.1
+   Host: trafficops.infra.ciab.test
+   User-Agent: curl/7.47.0
+   Accept: */*
+   Cookie: mojolicious=...
```

Comment:
okay, I see the problem here. These are indented with three spaces instead of a tab character, which is preventing it from rendering properly